lib/omniauth/strategies/instagram.rb

```diff
@@ -27,9 +27,15 @@ def request_phase
           'website'  => raw_info['website'],
         }
       end
+      
+      extra do
+        hash = {}
+        hash['raw_info'] = raw_info
+        hash
+      end
 
       def raw_info
-        @data ||= access_token.params["user"]
+        # @data ||= access_token.params["user"]
```

Comment:
1 year ago instagram return more data, when user authorised.
But now, it just return basic data, such name, nickname etc.
I need to know more info about user, such as followers count.
Method access_token.get('/v1/users/self') never been invoked, and I commented this code to fetch more info in "counts" hash key:

"data":  {
    "username": "pavel_galeta",
    "bio": "",
    "website": "",
    "profile_picture": "https://instagramimages-a.akamaihd.net/profiles/anonymousUser.jpg",
    "full_name": "pavel galeta",
    "counts":  {
      "media": 79,
      "followed_by": 56,
      "follows": 53
    },
    "id": "31973911"
  }